src/nft/components/bag/Bag.tsx

```diff
@@ -54,7 +52,9 @@ const ScrollingIndicator = ({ top, show }: SeparatorProps) => (
 )
 
 const Bag = () => {
-  const { account } = useWeb3React()
+  const { account, provider } = useWeb3React()
+  const isConnected = !!provider && !!account
```

Comment:
i think you could check for `isActive`?

Reply:
ah i guess that wouldn't work since it may be the network provider. this works then

Reply:
we don't do this anywhere currently, but I think we could use `Signer.isSigner(provider)`? https://docs.ethers.io/v5/api/signer/#Signer-isSigner

Reply:
we probably should do this lol. ill finishing reviewing now sorry got caught up